app/controllers/admin/merchants_controller.rb

```diff
@@ -4,4 +4,14 @@ def show
     @orders = Order.pending_orders(@merchant)
     redirect_to admin_user_path(@merchant) if @merchant.registered?
   end
+
+  def downgrade
```

Comment:
This is a good workaround for now, I'm fine with it. Great job

Reply:
We might be able to just call update on the merchant's items directly without doing the iteration, but I'm more than okay with this for now.